packages/osd-opensearch/package.json

```diff
@@ -12,7 +12,7 @@
     "osd:watch": "../../scripts/use_node scripts/build --watch"
   },
   "dependencies": {
-    "@opensearch-project/opensearch": "^2.6.0",
+    "@opensearch-project/opensearch": "^2.9.0",
```

Comment:
Nit: Is it necessary we upgrade the client version?

Reply:
Yeah. We need the version that uses JSON11